app/models/navigation_link.rb

```diff
@@ -0,0 +1,8 @@
+class NavigationLink < ApplicationRecord
```

Comment:
this is missing one validator: `validates :requires_auth, inclusion: { in: [true, false]}`

Learned recently that is important :D

Reply:
✅